### DISCUSSION TITLE
Ability to get notified about traffic spikes on a time period

### DISCUSSION THREAD

**Discussion by RobertJoonas** [plausible/analytics#1295] [Anything Else]
Currently, traffic spike notifications are only sent based on the real-time number of visitors. 

> "It would be great to get notified if a traffic spike occurred within a given time frame/period rather than exactly at the same time, which would be more realistic.
> 
> I.e., 200 visitors in the last 30 mins vs. 6 should trigger a traffic spike."

(feedback via email)

**Comment by ukutaht** [comment]
Should both the number of visitors *and* the time period be configurable?

**Comment by RobertJoonas** [comment]
A similar request via email regarding custom events:

> Would be nice to have the ability to set specific alerts on custom events.
> For example: "send me an email when a specific PDF is downloaded more than 100 times in one day"

Just thought I'd post it here, as perhaps we can make a more customizable section for alerts/notifications. What do you think @ukutaht?

**Reply by ukutaht** [reply]
Yeah, let's keep collecting the use-cases here so we can make something more flexible in the future
